mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### New features
+
+* Add `mp-fstab` tag to the User-Agent header when Mountpoint is launched via an fstab entry. This enables detection and analysis of fstab-based usage in product analytics. ([#1420](https://github.com/awslabs/mountpoint-s3/pull/1420))
```

Comment:
This is a minor analytics change, and doesn't warrant a change log entry IMO. If it did, I'd recommend putting it under `### Other changes` anyway since this isn't a new feature that would encourage customers to upgrade.